torch/distributed/pipeline/sync/batchnorm.py

```diff
@@ -151,8 +151,8 @@ def convert_deferred_batch_norm(cls, module: TModule, chunks: int = 1) -> TModul
             if module.affine:
                 module_output.register_parameter("weight", module.weight)
                 module_output.register_parameter("bias", module.bias)
-            assert isinstance(module.running_mean, Tensor)
-            assert isinstance(module.running_var, Tensor)
+            module.running_mean = cast(module.running_mean, Tensor)
+            module.running_var = cast(module.running_var, Tensor)
```

Comment:
Just curious. Why not also change this:
https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/batchnorm.py#L718-L719